std/net/curl.d

```diff
@@ -627,6 +627,61 @@ unittest
     assert(res == cast(ubyte[])[17, 27, 35, 41]);
 }
 
+/* Encode associative array using www-form-urlencoding
+ * perhaps this should live in std.uri
+ */
+private string urlencode()(in string[string] values)
+{
+    bool first = true;
+    Appender!string enc;
+    enc.reserve(values.length * 512);
+    foreach (k, v; values)
+    {
+        if (!first)
+            enc ~= '&';
+        import std.uri;
+
+        enc ~= encodeComponent(k);
+        enc ~= "=";
+        enc ~= encodeComponent(v);
+        first = false;
+    }
+    return enc.data;
+}
+
+/** HTTP post content.
+ *
+ * Params:
+ * url = resource to post to
+ * postData = data to send as the body of the request. An associative array of $(D string) is accepted and will be
+ *            encoded using www-form-urlencoding
+ * conn = HTTP connection to use
+ *
+ * Example:
```

Comment:
It's your call actually but it's been a moment now since people rather use ddoc unittest rather than plain ddoc examples sections. Excepted this detail this PR is fine now.